packages/standard-changelog/index.js

```diff
@@ -26,7 +26,7 @@ conventionalChangelog.createIfMissing = function (infile) {
 }
 
 conventionalChangelog.checkpoint = function (msg, args) {
-  console.info(chalk.green(figures.tick) + ' ' + sprintf(msg, args.map(function (arg) {
+  console.info(chalk.green(figures.tick) + ' ' + format(msg, ...args.map(function (arg) {
```

Comment:
alterantively this can be done without format as console.info support sprintf dirrectly
```suggestion
  console.info(chalk.green(figures.tick) + ' ' + msg, ...args.map(function (arg) {
```